### PR TITLE
feat(apig): add resource to manage vpc endpoint connnection of instance

### PR DESCRIPTION
**Pull request by wuzhuanhong**
**What this PR does / why we need it**:

Add new resource to manage VPC endpoint connnection under specified APIG instance.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. add new resource to manage VPC endpoint connnection under specified APIG instance.
2. add corresponding document and acceptance test.
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
 make testacc TEST=./huaweicloud/services/acceptance/apig TESTARGS='-run TestAccEndpointConnectionManagement_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/apig -v -run TestAccEndpointConnectionManagement_basic -timeout 360m -parallel 4
=== RUN   TestAccEndpointConnectionManagement_basic
=== PAUSE TestAccEndpointConnectionManagement_basic
=== CONT  TestAccEndpointConnectionManagement_basic
--- PASS: TestAccEndpointConnectionManagement_basic (646.26s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/apig      646.344s

```

### CODE REVIEW

**Comment by wuzhuanhong** [comment]
make testacc TEST=./huaweicloud/services/acceptance/apig TESTARGS='-run TestAccEndpointConnectionManagement_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/apig -v -run TestAccEndpointConnectionManagement_basic -timeout 360m -parallel 4
=== RUN   TestAccEndpointConnectionManagement_basic
=== PAUSE TestAccEndpointConnectionManagement_basic
=== CONT  TestAccEndpointConnectionManagement_basic
--- PASS: TestAccEndpointConnectionManagement_basic (631.53s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/apig      631.579s

File: huaweicloud/services/apig/resource_huaweicloud_apig_endpoint_connection_management.go [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+			"endpoints": []string{endpointId},
+		},
+	}
+	_, err := client.Request("POST", path, &opts)
+	if err != nil {
+		actionStatus := map[string]string{
+			"receive": "receiving",
+			"reject":  "rejecting",
+		}
+		return fmt.Errorf("error %s endpoint connection (%s) under dedicated instance (%s): %s", actionStatus[action], endpointId, instanceId, err)
```
Comment (Lance52259) [inline huaweicloud/services/apig/resource_huaweicloud_apig_endpoint_connection_management.go]:
```suggestion
		return fmt.Errorf("failed to %s endpoint connection (%s) under dedicated instance (%s): %s", actionStatus[action], endpointId, instanceId, err)
```
please using `failed to do sth` to instead, if you want to use the `error` to describe it, that it is the `error doing sth`.

Comment (wuzhuanhong) [reply huaweicloud/services/apig/resource_huaweicloud_apig_endpoint_connection_management.go]: fixed
image

**Comment by wuzhuanhong** [comment]
make testacc TEST=./huaweicloud/services/acceptance/apig TESTARGS='-run TestAccEndpointConnectionManagement_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/apig -v -run TestAccEndpointConnectionManagement_basic -timeout 360m -parallel 4
=== RUN   TestAccEndpointConnectionManagement_basic
=== PAUSE TestAccEndpointConnectionManagement_basic
=== CONT  TestAccEndpointConnectionManagement_basic
--- PASS: TestAccEndpointConnectionManagement_basic (627.59s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/apig      627.653s

**Pull request merged**